modules/test/playwright/tests/export-import-web/instance.import.spec.ts

```diff
@@ -122,6 +123,81 @@ test('can export and import custom object entries at instance level', async ({
 	);
 });
 
+test('can only import custom object entries when their definitions are already in the system', async ({
+	apiHelpers,
+	companyExportImportPage,
+}) => {
+	await companyExportImportPage.applicationsMenuPage.goToImport();
+	await companyExportImportPage.exportImportPage.newImportButton.click();
+
+	const compressedFile = await zipFolder(
+		path.join(__dirname, 'dependencies', 'test.object.portlet.lar')
```

Comment:
I don't think this is a good pattern for the input data, as it is very difficult to update it if we have to increase the coverage using this test, and in following versions, the LAR file does not need to be backwards compatible; so if we update the structure of the LAR file, we should need to update all these tests too, what it could become a nightmare.

I talked to Alex and both agrees on first exporting the data and then importing with our use case